tools/trim_galore/trim_galore.xml

```diff
@@ -48,8 +48,8 @@
         <requirement type="package" version="1.8.3">cutadapt</requirement>
         <requirement type="package" version="1.8">cutadapt</requirement>
     </requirements>
-    <version_command>
-        perl $__tool_directory/trim_galore --version
```

Comment:
I'd have to double check. I saw a few instances of it returning `perl $__tool_directory/...` as the version string, which seemed rather odd. Perhaps I had mucked something up, though. I'll test real quick.